packages/ui/v2/core/Shell.tsx

```diff
@@ -855,35 +851,17 @@ export function ShellMain(props: LayoutProps) {
   );
 }
 
-const SettingsSidebarContainerDefault = () => null;
-
 function MainContainer({
-  SettingsSidebarContainer: SettingsSidebarContainerProp = <SettingsSidebarContainerDefault />,
```

Comment:
No custom settingssidebar anymore, it is injected via the default SidebarContainer prop.